IfcMechanicalFastener Rail Import BIMQ 2021-07-21T07:45:27.4452357Z (#189)
bSI-InfraRoom/IFC-Specification: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MechanicalFastenerTypeRailJoint/Documentation.md

> Properties that can be used by every rail joint of railway track system.  The property set can be used by the predefined type RAILJOINT of IfcMechanicalFastener

Reviewer: We have speficy the applicable type in every documentation.

> Properties that can be used by every rail joint of railway track system.  The property set can be used by the predefined type RAILJOINT of IfcMechanicalFastener

Reviewer: specified*

> Properties that can be used by every rail joint of railway track system.  The property set can be used by the predefined type RAILJOINT of IfcMechanicalFastener

Reviewer: From the other reviewer, it's better to keep applicability into the Pset doc

> Properties that can be used by every rail joint of railway track system.  The property set can be used by the predefined type RAILJOINT of IfcMechanicalFastener

Reviewer 2: ```suggestion
Properties that can be used by every rail joint of railway track system.  The property set can be used by the predefined type RAILJOINT of IfcMechanicalFastener.
```

Reviewer: de4bcf9de7f5e2e1d9e24e66e3b338da3b5506eb

> Properties that can be used by every rail joint of railway track system.  The property set can be used by the predefined type RAILJOINT of IfcMechanicalFastener.

Reviewer 3: ```suggestion
Properties common to a rail joint of a railway track system.  The property set can be used by the predefined type RAILJOINT of IfcMechanicalFastener.
```
